### DISCUSSION TITLE
Can i interrupt sanoid/syncoid sync on command line and then resume later without issues?

### DISCUSSION THREAD

**Discussion by slrslr** [jimsalterjrs/sanoid#773] [Q&A] [answered]
$ syncoid -r sourcepool destinationpool
`INFO: Sending oldest full snapshot ... to new target filesystem: ...`

When i do Ctrl+C it intrerupts/ends the process.

Can i then later run same command again to resume without any data corruption and without copying data that was already copied in previous run?

**Comment by phreaker0** [comment, marked as answer]
Yes and Yes (if you have a recent version of ZFS)

**Reply by slrslr** [reply]
Do you know what is oldest ZFS version from which resume began working or a command to discover if it works not knowing the version? Mine:

> $ zfs --version
> zfs-2.1.5-1~bpo11+1
> zfs-kmod-2.1.5-1~bpo11+1

> $ man zfs-send|grep resume
>      zfs send [-Penv] -t receive_resume_token
>      zfs send [-Penv] -t receive_resume_token
>        Creates a send stream which resumes an interrupted receive.  The receive_resume_token is the value of this property on the filesystem or volume that was being received into.  See the documentation for zfs receive -s for
>        redaction can be resumed by rerunning the same command.

I am already numerous hours in transfer progress and do not want to break it by terminating it, so i can resume doing "syncoid --skip--parent -r src dest". My current sync command is "syncoid -r src dest"

**Reply by slrslr** [reply]
mine mentioned version appears to be able to resume..
